modules/installation-guide/examples/system-variables.adoc

```diff
@@ -29,23 +29,23 @@ pass:[<!-- vale off -->]
  `+CHE_WORKSPACE_ACTIVITY__CHECK__SCHEDULER__DELAY__S+`,"`+180+`","Delay before first workspace idleness check job started to avoid mass suspend if ws master was unavailable for period close to inactivity timeout." 
- `+CHE_WORKSPACE_CLEANUP__TEMPORARY__INITIAL__DELAY__MIN+`,"`+5+`","Period of stopped temporary workspaces cleanup job execution." 
- `+CHE_WORKSPACE_CLEANUP__TEMPORARY__PERIOD__MIN+`,"`+180+`","Periodof stopped temporary workspaces cleanup job execution." 
- `+CHE_WORKSPACE_SERVER_PING__SUCCESS__THRESHOLD+`,"`+1+`","Number of sequential successful pings to server after which it is treated as available. Note: the property is common for all servers e.g. workspace agent, terminal, exec etc." 
+ `+CHE_WORKSPACE_CLEANUP__TEMPORARY__INITIAL__DELAY__MIN+`,"`+5+`","Time period to delay the first execution of temporary workspaces cleanup job." 
+ `+CHE_WORKSPACE_CLEANUP__TEMPORARY__PERIOD__MIN+`,"`+180+`","Time period delay between the termination of one execution and the commencement of the next execution of temporary workspaces cleanup job" 
+ `+CHE_WORKSPACE_SERVER_PING__SUCCESS__THRESHOLD+`,"`+1+`","Number of sequential successful pings to server after which it is treated as available. the {prod-short} Operator: the property is common for all servers, for example, workspace agent, terminal, exec." 
  `+CHE_WORKSPACE_SERVER_PING__INTERVAL__MILLISECONDS+`,"`+3000+`","Interval, in milliseconds, between successive pings to workspace server." 
  `+CHE_WORKSPACE_SERVER_LIVENESS__PROBES+`,"`+wsagent/http,exec-agent/http,terminal,theia,jupyter,dirigible,cloud-shell,intellij+`","List of servers names which require liveness probes" 
  `+CHE_WORKSPACE_STARTUP__DEBUG__LOG__LIMIT__BYTES+`,"`+10485760+`","Limit size of the logs collected from single container that can be observed by che-server when debugging workspace startup. default 10MB=10485760" 
  `+CHE_WORKSPACE_STOP_ROLE_ENABLED+`,"`+true+`","If true, 'stop-workspace' role with the edit privileges will be granted to the 'che' ServiceAccount if OpenShift OAuth is enabled. This configuration is mainly required for workspace idling when the OpenShift OAuth is enabled." 
- `+CHE_DEVWORKSPACES_ENABLED+`,"`+false+`","Specifies whether che is deployed with DevWorkspaces enabled. This property is set by the {prod-short} operator if it also installed the support for DevWorkspaces. This property is used to advertise this fact to the {prod-short} dashboard. It does not make sense to change the value of this property manually." 
+ `+CHE_DEVWORKSPACES_ENABLED+`,"`+false+`","Specifies whether {prod-short} is deployed with DevWorkspaces enabled. This property is set by the {prod-short} Operator if it also installed the support for DevWorkspaces. This property is used to advertise this fact to the {prod-short} dashboard. It does not make sense to change the value of this property manually." 
```

Comment:
`Set by the {prod-short} Operator if support for DevWorkspaces is enabled - only used to identify DevWorkspaces support in the dashboard. There is no reason to override this property manually.`